_posts/2022-02-03_GitHook_to_clean_notebook.md

```diff
@@ -0,0 +1,33 @@
+# Using Git Hook to Clean Jupyter Notebook on Commit
+
+To avoid merge issues and make PR review easier, we wanted a [Git Hook](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks) which will:
+- Clear output of Jupyter Notebook
+- Only the files that were modified for this PR
```

Comment:
I think this should maybe read "Only {change|clean} the files that were modified for this PR"

Reply:
Ah yes